audio_extraction/audio_extraction.ipynb

```diff
@@ -0,0 +1,245 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ffmpeg\n",
+    "from pathlib import Path\n",
+    "import re\n",
+    "import logging\n",
+    "\n",
+    "# This makes jupyter notebook output everything instead of just the last output\n",
+    "from IPython.core.interactiveshell import InteractiveShell\n",
+    "InteractiveShell.ast_node_interactivity = \"all\"\n",
+    "\n",
+    "################################################################################\n",
+    "\n",
+    "# Considerations:\n",
+    "# Not sure if we should make the directories for the users if the supplied directory doesn't\n",
+    "# exist yet"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'data' folder exists!\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print error if this directory does not have a folder named 'data'\n",
```

Comment:
I've changed the code involving the data folder to be more concise.
```
# Make the data folder if it doesn't already exist
    Path(video_path).mkdir(exist_ok = True)
```
I've also added details about what folders get created when the program is run in the new readme file.

```
The video files should be in a folder named "data". The "data" folder should be in the same directory that contains `audio_extraction.py`.

### Directories
By default, `audio_extraction.py` reads video files from the "data" folder. This can be changed by changing the `video_path` argument in the `extract_audio_from_video` call.
```